oshc/main/templates/index.html

```diff
@@ -45,7 +45,7 @@ <h3 class="panel-title">{{ session.title }}</h3>
 						<div class="line-gap-10px"></div>
 						<i>{{ session.start_date }} - {{ session.end_date }}</i>
 						<div class="line-gap-10px"></div>
-						<button type="button" class="btn btn-lg btn-primary sRegister">
+						<button type="button" class="btn btn-lg btn-primary s">
```

Comment:
extraneous change. @abhishekarya286 Please don't change anything that are not relevant to the issue.

Reply:
Sorry, I did this by mistake. I have edited it now.

Reply:
@abhishekarya286 It's ok to make mistakes :wink: 
One learn from his mistakes